src/core/internal/dassert.d

```diff
@@ -383,3 +398,33 @@ private auto pureAlloc(size_t t)
     }
     return assumeFakeAttributes(&alloc)(t);
 }
+
+// https://issues.dlang.org/show_bug.cgi?id=21544
+unittest
+{
+    // Normal enum values
+    enum E { A, BCDE }
+    E e = E.A;
+    assert(miniFormat(e) == "A");
+    e = E.BCDE;
+    assert(miniFormat(e) == "BCDE");
+
+    // Invalid enum value is printed as their implicit base type (int)
+    e = cast(E) 3;
+    assert(miniFormat(e) == "cast(E)  3");
```

Comment:
Why the two spaces ?

Reply:
`combine` inserts a space before and after the token (here `""`)